packages/webpack-extraction-plugin/src/babelPluginStripGriffelRuntime.ts

```diff
@@ -24,6 +26,170 @@ function concatCSSRulesByBucket(bucketA: CSSRulesByBucket = {}, bucketB: CSSRule
   return bucketA;
 }
 
+function evaluateAndUpdateArgument(
+  argumentPath: NodePath<t.ObjectExpression> | NodePath<t.ArrayExpression>,
+  functionKind: FunctionKinds,
+  state: StripRuntimeBabelPluginState,
+) {
+  // Returns the styles as a JavaScript object
+  const evaluationResult = argumentPath.evaluate();
+
+  if (!evaluationResult.confident) {
+    throw argumentPath.buildCodeFrameError(
+      [
+        `Failed to evaluate CSS rules from "${functionKind}" call.`,
+        'Please report a bug (https://github.com/microsoft/griffel/issues) if this error happens',
+      ].join(' '),
+    );
+  }
+
+  if (functionKind === '__styles') {
+    const cssRulesByBucket = evaluationResult.value as CSSRulesByBucket;
+
+    state.cssRulesByBucket = concatCSSRulesByBucket(state.cssRulesByBucket, cssRulesByBucket);
+  } else if (functionKind === '__resetStyles') {
+    const cssRules = evaluationResult.value as NonNullable<CSSRulesByBucket['r']>;
+
+    state.cssRulesByBucket = concatCSSRulesByBucket(state.cssRulesByBucket, { r: cssRules });
+  }
+
+  argumentPath.remove();
+}
+
+function getFunctionArgumentPath(
+  callExpressionPath: NodePath<t.CallExpression>,
+  functionKind: FunctionKinds,
+): NodePath<t.ObjectExpression> | NodePath<t.ArrayExpression> | null {
+  const argumentPaths = callExpressionPath.get('arguments');
+
+  if (
+    (functionKind === '__styles' && argumentPaths.length !== 2) ||
+    (functionKind === '__resetStyles' && argumentPaths.length !== 3)
+  ) {
+    const calleePath = callExpressionPath.get('callee');
+
+    throw calleePath.buildCodeFrameError(
+      [
+        `"${functionKind}" function call should have exactly ${argumentPaths.length} arguments.`,
```

Comment:
does this error make sense?

* call `__styles` with 3 arguments
* '__styles shoud have exactly **3** arguments'

Reply:
Nope and this error should be removed at all: `__styles` can be used in another packages and this could throw. I replaced it with other checks.

Reply:
yeah the existing checks already cover that pretty well